ingot-macros/src/choice.rs

```diff
@@ -291,6 +291,14 @@ pub fn attr_impl(attr: TokenStream, item: syn::ItemEnum) -> TokenStream {
             }
         }
 
+        impl<V: ::ingot::types::SplitByteSlice> ::ingot::types::ToOwnedPacket for #validated_ident<V> {
+            type Target = #repr_head;
+            #[inline]
+            fn to_owned(&self, _hint: ::core::option::Option<Self::Denom>) -> ::ingot::types::ParseResult<Self::Target> {
```

Comment:
I feel like this should be `Self::Hint`, but that fails to build `ingot-examples`, so I'm digging further...